### PR TITLE
add_dependency "rest-client"

### PR DESCRIPTION
**Pull request by debbbbie**
`add_development_dependency "rest-client"` => `add_dependency "rest-client"`

不然项目Gemfile 里还要手动添加 `gem 'rest-client'`

### CODE REVIEW

File: lib/generators/cloopen/templates/cloopen_setup.rb [hunk 1]
```diff
@@ -2,13 +2,13 @@
 
 Cloopen.account_sid = "YOUR_ACCOUNT_SID"
 Cloopen.auth_token  = "YOUR_AUTH_TOKEN"
-Cloopen.app_id = "YOUR_APP_ID"
-Cloopen.env = "production"
+Cloopen.app_id      = "YOUR_APP_ID"
+Cloopen.env         = "production"
```
Comment (debbbbie) [inline lib/generators/cloopen/templates/cloopen_setup.rb]: I think that will be beautiful. :smile: @houndci

Comment (debbbbie) [inline lib/generators/cloopen/templates/cloopen_setup.rb]: Need I remove it?

Comment (xiaoronglv) [inline lib/generators/cloopen/templates/cloopen_setup.rb]: No.


I already merge your pull request, and released the new version.


Great work, Thanks.


-- 
Ryan, Lv


On June 6, 2015 at 11:56:19, debbbbie (notifications@github.com) wrote:

In lib/generators/cloopen/templates/cloopen_setup.rb:

> @@ -2,13 +2,13 @@
>   
>  Cloopen.account_sid = "YOUR_ACCOUNT_SID"
>  Cloopen.auth_token  = "YOUR_AUTH_TOKEN"
> -Cloopen.app_id = "YOUR_APP_ID"
> -Cloopen.env = "production"
> +Cloopen.app_id      = "YOUR_APP_ID"
> +Cloopen.env         = "production"
Need I remove it?

Comment (debbbbie) [inline lib/generators/cloopen/templates/cloopen_setup.rb]: :+1:

**Pull request merged**